.github/workflows/scorecard.yml

```diff
@@ -32,12 +33,12 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
I thought the decision was made to not pin actions from GitHub itself, has this been changed?

Reply:
I wasn’t aware of that decision. The Pinned Dependencies check is classified as a Medium threat according to the Scorecard checks ([reference](https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies)).

@jonkoops Could you please share the documentation or context regarding where this decision was made?

Reply:
I remember it was being discussed, but it seems to be formalized now, so consider this resolved.